apps/explorer/lib/explorer/counters/token_holders_counter.ex

```diff
@@ -65,21 +65,33 @@ defmodule Explorer.Counters.TokenHoldersCounter do
 
   defp update_cache(address_hash) do
     address_hash_string = to_string(address_hash)
-    put_into_cache("hash_#{address_hash_string}_#{@last_update_key}", Helper.current_time())
-    new_data = Chain.count_token_holders_from_token_hash(address_hash)
-    put_into_cache("hash_#{address_hash_string}", new_data)
+    Helper.put_into_ets_cache(@cache_name, "hash_#{address_hash_string}_#{@ets_last_update_key}", Helper.current_time())
+    new_data = CurrentTokenBalance.count_token_holders_from_token_hash(address_hash)
+    Helper.put_into_ets_cache(@cache_name, "hash_#{address_hash_string}", new_data)
```

Comment:
Shouldn't we put 'updated_at' after `count_token_holders_from_token_hash` completion?
```suggestion
    new_data = CurrentTokenBalance.count_token_holders_from_token_hash(address_hash)
    Helper.put_into_ets_cache(@cache_name, "hash_#{address_hash_string}", new_data)
    Helper.put_into_ets_cache(@cache_name, "hash_#{address_hash_string}_#{@ets_last_update_key}", Helper.current_time())
```